version: '3.5'
services:
  home-assistant:
    build: ./home-assistant/
    extra_hosts:
      - "outside:${HOME_ASSISTANT_IP}"
    env_file:
      - '.env.home-assistant'
      - '.env.influxdb'
      - '.env.local'
    restart: unless-stopped
    cap_add:
      - net_bind_service
    privileged: true
    network_mode: host
    environment:
      - TZ=Europe/Warsaw
      - HOME_ASSISTANT_DB_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:4101/${POSTGRES_DB}
    volumes:
      - ./home-assistant:/config
      - /var/log/fail2ban.log:/fail2ban/fail2ban.log:ro
      - /tmp:/tmp
    devices:
      - ${HOME_ASSISTANT_ZWAVE_DEV}:/dev/ttyACM0
  appdaemon:
    image: acockburn/appdaemon:latest
    restart: unless-stopped
    network_mode: host
    env_file:
      - '.env.home-assistant'
    environment:
      - HA_URL=http://localhost:8123
      - TOKEN=${HOME_ASSISTANT_API_PASSWORD}
      - DASH_URL=http://localhost:15050
    volumes:
      - ./appdaemon:/conf
  busix:
    image: macbury/busix:latest
    network_mode: host
    env_file:
      - '.env.busix'

